docs/usage/configuration-reference.md

```diff
@@ -61,43 +62,40 @@ The structure object defines the directory structure where Shunter's resources w
 
 ```js
 structure: {
-    templates: 'view',
-    templateExt: '.dust',
-    resources: 'resources',
-    styles: 'css',
-    images: 'img',
-    scripts: 'js',
-    fonts: 'fonts',
-    tests: 'tests',
+    dust: 'dust',
+    ejs: 'ejs',
     filters: 'filters',
     filtersInput: 'input',
     filtersOutput: 'output',
-    dust: 'dust',
-    ejs: 'ejs'
+    fonts: 'fonts',
+    images: 'img',
+    logging: 'logging',
+    loggingTransports: 'transports',
+    loggingFilters: 'filters',
```

Comment:
I would like to refactor this so that `logging` & `filters` and maybe `templates` are their own sub-objects, but that can wait...